### PR TITLE
Integrate an example without content coding.

### PR DESCRIPTION
**Pull request by ioggstream**
## This PR

adds an example without content coding

### CODE REVIEW

File: draft-ietf-httpbis-digest-headers.md [hunk 1]
```diff
@@ -233,7 +233,20 @@ and method impacts on the message and payload body. When the payload body
 contains non-printable characters (eg. when it is compressed) it is shown as
 base64-encoded string.
 
+A simple request with a json object without any content coding
```
Comment (LPardue) [inline draft-ietf-httpbis-digest-headers.md]:
```suggestion
A request with a json object without any content coding
```

**Comment by ioggstream** [comment]
Do we still need it?

**Comment by LPardue** [comment]
The change affects a section that was moved toward the end of the document. So rebasing will address my earlier comment. I see the utility in the example but I'll leave it up to you if you want to merge or not.

**Comment by ioggstream** [comment]
@LPardue we moved this part below, so merge is straightforward. Feel free to press the button :)

**Comment by LPardue** [comment]
cheers!

**Pull request merged**